common/authen.js

```diff
@@ -240,49 +268,78 @@
             logInHelper(loginWindowCb, win, reloadCb, 'popUp', null, logAction);
         };
 
-        var shouldReloadPageAfterLogin = function(newSession) {
-            if (_session === null) return true;
+        var shouldReloadPageAfterLogin = function() {
```

Comment:
Can you please add a comment to explain the cases that we're actually reloading after login? it's a bit confusing right now and you have to follow the code path to understand it.